tests/check_pvt.c

```diff
@@ -86,6 +86,8 @@ START_TEST(test_pvt_repair)
   s8 code = calc_PVT(n_used, nms, false, &soln, &dops);
   fail_unless(code == 1,
     "Return code should be 1 (pvt repair). Saw: %d\n", code);
+  fail_unless(soln.raim_flag == 3,
+    "raim_flag should be 11, saw: %d\n", soln.raim_flag);
```

Comment:
confusing.. prefix the '11' with 0b or otherwise clarify that the "should be" and "saw" are different bases